app/views/model_mailer/new_rent_notification.html.slim

```diff
@@ -1,12 +1,12 @@
 h1
-  | New rent created
+  = t('new_rent')
 h2
-  | Created at: #{ @greeting.created_at } 
+  | #{ I18n.t 'created_at' } : #{ @greeting.created_at }
```

Comment:
`I18n.t()` and `t()` are alias, but the shorter form is the preferred one (remember to use it always with parentheses).
`t('created_at')`